astropy/table/meta.py

```diff
@@ -47,105 +45,6 @@ def items(self):
         return ColumnOrderList(super().items())
 
 
-def _construct_odict(load, node):
```

Comment:
Is this OK? If someone has an old yaml file, will it still load?

Reply:
I think so? There was a failing test trying to load something in `coordinates` but it was fixed with the weird patches I pointed out above.